yarn.lock

```diff
@@ -3329,7 +3329,7 @@ gopd@^1.0.1:
   dependencies:
     get-intrinsic "^1.1.3"
 
-govuk-frontend@^4.6.0:
+govuk-frontend@^4.7.0:
   version "4.7.0"
```

Comment:
Was dependabot bugged?  This line says 4.7.0 already despite the line above saying 4.6.0?!

Reply:
More likely our lock file, but this should now enable future bumping.